crates/cargo-rm/src/bin/cargo/commands/rm.rs

```diff
@@ -0,0 +1,105 @@
+use cargo::{core::dependency::DepKind, util::command_prelude::*};
+use cargo_rm::ops::cargo_rm::{rm, DepTable, RmOptions};
+
+pub fn cli() -> clap::Command<'static> {
+    clap::Command::new("rm")
+        .setting(clap::AppSettings::DeriveDisplayOrder)
+        .about("Remove dependencies from a Cargo.toml manifest file")
+        .args([
```

Comment:
Refactoring from derive to builder should be done in a separate PR.

This PR should just be about moving files.  Anything else will be impossible to review.

Reply:
Oh, huh. I didn't see this sort of refactor happening for cargo add. This cli rewrite involved updating to clap latest to be in line with cargo, so should I update it on master as well? That will have consequences for all of the other binaries as well, including cargo add.

Reply:
I've gone ahead and updated clap in master: https://github.com/killercup/cargo-edit/pull/769

Reply:
Okay! I'll refactor to builder now.

Reply:
When working towards atomic commits, when moving code, you only move the code and update references.  Another commit then handles porting the way the code works.

For example, merge-add:
- [Pulling out a data structure for assign arguments](https://github.com/killercup/cargo-edit/commit/3a4866318b8185dbb7a1621b73b6d3135a8d6be9)
- [Moving one half of the code path for that data structure into an op](https://github.com/killercup/cargo-edit/commit/75c2b5959c0e15083dab450434e6800a473f698d)

Not finding it atm the moment but I believe merge-add did a complete port of the derive CLI to the builder CLI and then integrated it into cargo's pattern.  It takes some more thinking, planning, and sometimes throwing things away, but that is the cost of clear communication.

Reply:
I've merged my change into master and merge-rm